core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -297,79 +301,80 @@ oppia.factory('oppiaPlayerService', [
         }
 
         answerIsBeingProcessed = true;
-        var oldState = exploration.getState(
-          playerTranscriptService.getLastStateName());
+        var stateName = playerTranscriptService.getLastStateName();
+        var oldState = exploration.getState(stateName);
         AnswerClassificationService.getMatchingClassificationResult(
-          _explorationId, oldState, answer, false, interactionRulesService
-        ).then(function(classificationResult) {
-          if (!_editorPreviewMode) {
-            StatsReportingService.recordAnswerSubmitted(
-              playerTranscriptService.getLastStateName(),
-              LearnerParamsService.getAllParams(),
-              answer,
-              classificationResult.answerGroupIndex,
-              classificationResult.ruleIndex,
-              classificationResult.classificationCategorization);
-          }
+          _explorationId, stateName, oldState, answer, false,
+          interactionRulesService)
+          .then(function(classificationResult) {
```

Comment:
Bring the ')' from the previous line down to the start of this line (similar to how it was before), then you don't need all the indentation changes.